Gems/Multiplayer/Code/Include/Multiplayer/IMultiplayer.h

```diff
@@ -182,18 +187,20 @@ namespace Multiplayer
     class ScopedAlterTime final
     {
     public:
-        inline ScopedAlterTime(HostFrameId frameId, AZ::TimeMs timeMs, AzNetworking::ConnectionId connectionId)
+        inline ScopedAlterTime(HostFrameId frameId, AZ::TimeMs timeMs, float blendFactor, AzNetworking::ConnectionId connectionId)
         {
             INetworkTime* time = GetNetworkTime();
             m_previousHostFrameId = time->GetHostFrameId();
             m_previousHostTimeMs = time->GetHostTimeMs();
             m_previousRewindConnectionId = time->GetRewindingConnectionId();
             time->AlterTime(frameId, timeMs, connectionId);
+            time->AlterBlendFactor(blendFactor);
         }
         inline ~ScopedAlterTime()
         {
             INetworkTime* time = GetNetworkTime();
             time->AlterTime(m_previousHostFrameId, m_previousHostTimeMs, m_previousRewindConnectionId);
+            time->AlterBlendFactor(DefaultBlendFactor);
```

Comment:
Don't we want this to do the same snap off current blend factor and restore blend factor? As an example, nested scoped alter times are something that happens during backward reconciliation of projectiles.

Reply:
Ah okay, I hadn't realized we were nesting. I had thought it was more of an iterative thing. I'll check through the reconciliation code and make updates.